setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
```

Comment:
is this file necessary? I thought it was replace by pyproject.toml

Reply:
I put in your mind the idea that setup.py was deprecated.
However I found this link
https://packaging.python.org/en/latest/discussions/setup-py-deprecated/

**Should setup.py be deleted?**

Although the usage of setup.py as an executable script is deprecated, its usage as a configuration file for setuptools is absolutely fine. There is likely no modification needed in setup.py.

**Is pyproject.toml mandatory?**
While it is not technically necessary yet, it is STRONGLY RECOMMENDED for a project to have a pyproject.toml file at the root of its source tree.

When creating the packet now configuration of setup.py is not read, I want to keep it for a while to confirm that we are not using at any time.

Reply:
ok!